V0.10 Masterlist Update - New Entries & Remove implicit USSEP bash tags

MacSplody opened the pull request:
Removed some implicit USSEP bash tags
- Tags removed were result of changes forwarded from Unofficial Skyrim Special Edition Patch, which was not an explicit master.

DLC Integration SE 
- Add Compatibility patch for Beyond Skyrim Bruma & warning message.
- Clean plugin info added for Compatibility patch
- Moved just below BS Skyrim entry for clarity, since I was adding the Compatibility patch.

Realistic Water Two
- Add bash tag Sound
- Clean plugin info

New Entry's Added
- Serana dialogue Edit
- The Choice is Yours
- DLC Integration Beyond Skyrim Compatibility Patch SE
- RLO - Latest version uses different naming scheme.
- Unlimited bookshelves.

Navigation
- Add Comment where Mod name was not clear from plug-in name.

Cleanup
- indentation fixes
- Replace warn messages with *hasPluginPatch where possible.

MacSplody committed:
Remove Implicit USSEP Tags

- Tags are inherited from changes forwarded from USSEP, even though USSEP is not a master.

MacSplody committed:
New Entry - DLC Intergration Patch

DLCIntegration.esp
- Added compatibilty patch & msg

DLCIntegration Beyond Skyrim Compatibility Patch
- Clean plugin info

BS_DLC_patch.esp
- Remove warning message about DLCintegration compatibility.

MacSplody committed:
Alter subs entry name

MacSplody committed:
Update entries

SMIM
- No static mesh improvement mod se.esp file available.
- Replaced with BSA version.
- SMIM-SE-.* still covers all esp in modular version.
- Clean plugin info for SMIM BSA version 2.07

Bash tag updates
- Remove C.Water tag when entry was reverting Update.esm & USSEP changes.

MacSplody committed:
Bash tag Update

Morrowloot Ultimate
Undeath

MacSplody committed:
Mod Name fix

Ortham commented on masterlist.yaml:
```diff
@@ -545,20 +561,15 @@ plugins:
-      - Names
-      - Relev
-      - Stats
+    - Actors.ACBS
```
This list should be indented by another two spaces.

Ortham commented on masterlist.yaml:
```diff
@@ -576,8 +587,14 @@ plugins:
     clean:
       - crc: 0xB8A1EE4E
         util: SSEEdit v3.2.1
-  - name: 'static mesh improvement mod se.esp'
```
Was this wrong, or did the plugin name just change?

MacSplody replied:
It's possible that it was wrong, not sure if it was used before, but there is no plug-in listed with that name now.
smim-se-.*.esp check always and still works fine for manual version.

Plugins currently available for SE on NEXUS: Is it hosted on afk or anywhere else?

- SMIM-SE-FarmhouseFlickeringFix.esp
- SMIM-SE-SolitudeDocksFixes.esp
- SMIM-SE-DragonbornTernFix.esp
- SMIM-SE-ShackRoofFixes.esp
- SMIM-SE-ShackRoofFixesDragonborn.esp
- SMIM-SE-RiftenDocksRopesObjectFix.esp
- SMIM-SE-FurnitureChestSnowFix.esp
- SMIM-SE-DungeonsCliffsIceSkirts.esp
- SMIM-SE-Merged-All.esp
- SMIM-SE-Merged-NoRiftenRopes.esp
- SMIM-SE-Merged-NoRiftenRopes-NoSolitudeRopes.esp
- SMIM-SE-Merged-NoSolitudeRopes.esp
- SMIM-SE.esp

Ortham replied:
I don't know... I'd err on the side of adding another entry for SMIM-SE.esp to support both.

MacSplody replied:
Okay will restore that one and a new entry.

Ortham commented on masterlist.yaml:
```diff
@@ -804,6 +822,7 @@ plugins:
       - 'Verdant - A Skyrim Grass Plugin SSE Version.esp'
       - 'Immersive Citizens - AI Overhaul.esp'
   - name: 'RealisticWaterTwo.esp'
+    #global_priority: 72
```
Looks like you left this in by accident.

MacSplody replied:
yeah, will remove that.

Ortham commented on masterlist.yaml:
```diff
@@ -1245,15 +1260,6 @@ plugins:
   - name: 'TrueStormsSE.esp'
     tag:
       - Sound
-  - name: 'DLCIntegration.esp'
```
Please try to avoid moving entries around in future, it's confusing.

MacSplody replied:
I generally try to avoid it, it made sense in this case due to the type of mod. Personally I'd like to sort as many of them as possible into default groups.

Ortham replied:
> Personally I'd like to sort as many of them as possible into default groups.

The order of entries in the masterlist is meaningless and always will be, so it's just a cosmetic thing.

MacSplody committed:
Urls & Indentation fix

MLU patch message for
- Immersive armors
- Improved closeface helmets

Add url & comment with mod name where the plugin name did not match the mods name.

Fix Indentation on Bijin mods.

MacSplody committed:
Remove comment

MacSplody committed:
Indetation fix bash tags

MacSplody committed:
Add url & fix comment placement

MacSplody committed:
Remove "Patch" from has plugin subs

- Not needed as the word is all ready in main message

MacSplody committed:
New Entry - Serana Dialogue Edit

- Added Load after rules. Minor conflicts but will break the mod if either is loaded after it.

MacSplody committed:
Replace Plugin Patch warn message.

- Improved closefaced helmets for Undeath SSE Patch

MacSplody commented:
There are entries for Realistic Lighting Overhaul which are using the old plug-in names as well.
Those files are still available, but in the older file section. 
New files are named:
- RLO - Interiors.esp
- RLO - Exteriors
- RLO - Effects.esp
- RLO - Illuminated Spells.esp

Patches
- RLO - CRF Patch.esp
- RLO - IC Patch.esp
- RLO - JK's Riverwood.esp
- RLO - JK's Whiterun.esp
- RLO - VIS Patch.esp

I assume the old entries can stay as the files are still available.

Edit: Not much around in regards to RLO version 5 compatibility notes...

MacSplody committed:
Added The Choice is Yours

- Clean plugin info
- load after rules
- bash tags
- Patch available for book covers.

MacSplody committed:
Minor fixes

- Add version comment above crc
- Removed crc from clean for Undeath esp which still contained wild edits.
- fixed conditional & wording for deadly combat & ordinator patch.

CptMcSplody commented on masterlist.yaml:
```diff
@@ -185,6 +185,7 @@ globals:
 
 plugins:
+### Official Update & DLC plugins
   - name: 'Update.esm'
```
Same separators were in F04 master-list. Thought it would useful for when groups are added.

MacSplody commented on masterlist.yaml:
```diff
@@ -185,6 +185,7 @@ globals:
  
 
 plugins:
+### Official Update & DLC plugins
```
Same separators were in F04 master-list. Thought it would be useful for when groups are added.

Ortham replied:
Groups are already in the `v0.13` branch. I think you might be misunderstanding how they're used though? They're not like BOSS groups where they divide the masterlist up into sections, I anticipate they'll be used much more sparingly (they are just replacing priorities after all).

MacSplody replied:
Ah right, for some reason I thought they would be used as the default way to sort the load order with "load after" overriding them. So will I remove these separators?

Ortham replied:
They're more of a lower-priority extension to "load after". Removing the separators would probably be best.

MacSplody commented on masterlist.yaml:
```diff
@@ -395,29 +397,41 @@ plugins:
+    after:
+      - 'Lanterns Of Skyrim - All In One - Main.esm'
     clean:
+      # Version 4.1.2alpha
```
Again, something that was in the F04 master-list, it does add a bit of fluff though, if you have no issue with these I'll keep using them for maintenance.

Ortham replied:
No issue, I think they're helpful.

MacSplody commented on masterlist.yaml:
```diff
@@ -395,29 +397,41 @@ plugins:
+### Other Plugins
   - name: 'BSAssets.esm'
+    url: ['https://www.nexusmods.com/skyrimspecialedition/mods/10917/']
+    # Beyond Skyrim - Bruma SE
```
I added these in cases where the plug-in name did not have the mod name included.

Ortham replied:
Yep, that's helpful.

MacSplody commented on masterlist.yaml:
```diff
@@ -395,29 +397,41 @@ plugins:
-      - Graphics
+### Other Plugins
   - name: 'BSAssets.esm'
+    url: ['https://www.nexusmods.com/skyrimspecialedition/mods/10917/']
```
I know the URL are not strictly required, but it does make it easier for maintenance.

Ortham replied:
Yep 👍

MacSplody committed:
Restore SMIM entry

CptMcSplody commented on masterlist.yaml:
```diff
@@ -1050,6 +1050,15 @@ plugins:
+    url: ['https://www.nexusmods.com/skyrimspecialedition/mods/16222/']
+    after:
+      - 'Book Covers Skyrim.esp'
+      - 'Relationship Dialogue Overhaul.esp'
```
Set to load after both as per Mod Author instruction. There is a patch available for book covers but it will won't be needed after the next update, and it's only one texture.

MacSplody committed:
Remove Separators

MacSplody commented on masterlist.yaml:
```diff
@@ -1327,3 +1327,21 @@ plugins:
+      - 'BetterQuestObjectives.esp'
+      - 'Book Covers Skyrim.esp'
+      - 'Immersive Citizens - AI Overhaul.esp'
+      - 'Relationship Dialogue Overhaul.esp'
```
No major conflicts, but these mods will overwrite changes made by The choice is yours.

MacSplody committed:
Updated Entries

- Cutting room floor
- SMIM
- Touring Carriages

MacSplody committed:
Fix syntax

MacSplody committed:
Update Entries - SkyUI & Open Cities

Added
- clean plugin verification
- Urls

MacSplody commented on masterlist.yaml:
```diff
@@ -769,9 +769,15 @@ plugins:
       - *SKSE64
+    clean:
+      # Version 5.2
+      - crc: 0x79ACE179
```
Maybe a bit unnecessary as the plug-in is just a quest for running it.

MacSplody committed:
Updated - Even Better Quest Objectives

- url & name added
- Bash tag added
- Clean plugin info
- Patches are included in installer and added automatically.
- Not all patches have been ported, such as the one for Amazing Follower tweaks & Immersive AFT

MacSplody commented on masterlist.yaml:
```diff
@@ -715,8 +715,14 @@ plugins:
       - 'Realistic Lighting Overhaul - Major City Interiors.esp'
   - name: 'BetterQuestObjectives.esp'
+    url: ['https://www.nexusmods.com/skyrimspecialedition/mods/159/']
+    # Even Better Quest Objectives
```
Could the global priority for this mod be set to (-1)? As it just replaces quest entry names & objectives. Any mods it does conflict with it has a patch included or is easily patched. It doesn't have a patch included for Amazing follower tweaks or Immersive AFT which it will break if it overwrites. Or I could add a load after rule to those mods.

MacSplody committed:
Update existing entries, Added New RLO plugins, Unlimited Bookshelves

Add urls
Add Mod name

Ortham commented on masterlist.yaml:
```diff
@@ -1286,3 +1434,53 @@ plugins:
+      - crc: 0x86CF0F2B
+        util: SSEEdit v3.2.1
+  # Realistic Lighting Overhaul SE V5
+  - name: 'RLO - Effects.esp'
```
I don't think there's any point adding entries just containing a URL, since they don't actually do anything.

Ortham commented:
I'm going to close this PR because it's gotten too big for me to review effectively. I'd like to merge most of the changes it's making, but please split it into a series of smaller PRs with more limited scope.

A good general rule is that a PR shouldn't grow in scope once you've submitted it. If I request changes and you make them but then you also add more stuff on top that I'm then not happy with, I can't merge the bits you originally submitted that I'm now happy with, and we end up in a potentially endless feedback loop.

MacSplody commented:
Fair enough, I guess I just got a bit to eager with the commits. I'll add them in more manageable commits.
